pkg/reportingplugins/mercury/offchain_config.go

```diff
@@ -0,0 +1,20 @@
+package mercury
+
+import (
+	"encoding/json"
+)
+
+type OffchainConfig struct {
+	ExpirationWindow uint32 // Integer number of seconds
+	BaseUSDFeeCents  uint32
+}
+
+func DecodeOffchainConfig(b []byte) (o OffchainConfig, err error) {
+	// TODO: consider protobuf for better efficiency
```

Comment:
I would say let's remove this TODO since we are not planning on changing it for now